deploy-as-code/helm/charts/ifix-services/ifix-es-pipeline/values.yaml

```diff
@@ -25,4 +25,11 @@ env: |
         name: egov-config
         key: kafka-brokers
   - name: FISCAL_EVENT_KAFKA_PUSH_TOPIC
-    value: {{ index .Values "fiscal-event-kafka-push-topic" | quote }}
\ No newline at end of file
+    value: {{ index .Values "fiscal-event-kafka-push-topic" | quote }}
+  - name: IFIX_DEPARTMENT_ENTITY_SEARCH_ENDPOINT
+    value: {{ index .Values "ifix-department-entity-search-endpoint" | quote }}
+  - name: IFIX_DEPARTMENT_ENTITY_SERVICE_HOST
+    valueFrom:
+      configMapKeyRef:
+        name: egov-service-host
+        key: ifix-department-entity-service
```

Comment:
This [service](https://github.com/misdwss/iFix-DevOps/blob/ifix/deploy-as-code/helm/environments/ifix-dev.yaml#L59) has been moved to mgramseva namespace. It doesn't exist in ifix anymore. 

We had decided to move the department entity service from ifix to mgramseva. Now, if you are calling that service from ifix, then it should be present in ifix namespace. Please clarify and decide.

Reply:
I can still see department entity service in ifix namespace. Mgramseva has the up to date data, is it?

Reply:
Yes. The service in the mgramseva namespace has the latest data. We need to delete the service from ifix namespace.

Reply:
Updated host to point to mgramseva instance